README.md

```diff
@@ -68,7 +85,9 @@ docker build -t firecloud-orch .
 Run the container:
 ```
 docker run --name orch \
+  -p 8080:8080
+  -e GOOGLE_CLIENT_ID='$GOOGLE_CLIENT_ID' \
```

Comment:
Yup ... take out the single quotes around $GOOGLE_CLIENT_ID' and I'm able to see the swagger interface.